src/BuildIntegration/Microsoft.NETCore.Native.targets

```diff
@@ -222,7 +222,7 @@ See the LICENSE file in the project root for more information.
 
     <Message Text="Generating native code" Importance="high" />
 
-    <Exec Command="&quot;$(IlcPath)\tools\ilc&quot; @&quot;$(NativeIntermediateOutputPath)%(IlcCmd.ResponseFileName)&quot;" />
+    <Exec Command="&quot;$(IlcPath)\tools\ilc&quot; @&quot;$(NativeIntermediateOutputPath)%(IlcCmd.ResponseFileName)&quot;" IgnoreExitCode="true" />
```

Comment:
That's how BuildFrameworkNativeObjects.proj works in full AOT mode (I hit that very recently because two framework assemblies fail to full-AOT compile with a checked RyuJIT): first failure doesn't make the build stop - it continues AOT building the rest of the framework and then fails. If this is not happening for BuildReadyToRunAssemblies.proj, it's probably caused by one of the few lines where the files diverge.